README.md

```diff
@@ -1,62 +1,37 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+Hello Microverse
 
-> Description the project.
+> This is a project as part of a code review exercise.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+- HTML5, CSS3
+- github flow
+- Git action
 
 
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
+### Usage : You can download the zip file https://github.com/digia3d/Hello-world/tree/Hello-Microverse.
```

Comment:
- [ **OPTIONAL** ] I think it would be better to use the good syntax of Github when writing commands in your documentation to make it look more professional and smart. To do this you can wrap commands inside [**back ticks**](https://www.google.com/search?q=back+stick+symobol&oq=back+stick+symobol&aqs=chrome.0.69i59j0i10i19.3217j0j7&sourceid=chrome&ie=UTF-8) and it will output things like this:
`git clone https://github.com/digia3d/Hello-world/tree/Hello-Microverse` clone project
`cd Hello-Microverse` to access the cloned repository.